src/calendar/CalendarMonth.jsx

```diff
@@ -30,11 +24,39 @@ const CalendarMonth = React.createClass({
     onMonthChange: React.PropTypes.func,
     onYearChange: React.PropTypes.func,
     value: CustomPropTypes.momentOrMomentRange,
+    locale: React.PropTypes.string,
+  },
+
+  _setupLocale(locale) {
+    this.moment = moment();
+
+    if (locale !== 'en') {
+      require(`moment/locale/${locale}`);
```

Comment:
It's a problem with webpack/browserify. On the other hand, letting the user import the locale has issues of it's own, as I noticed running my patch in production:

``moment`` is a dependency of the datepicker. That means it is possible that we have our own copy of it in:

    ``project_dir/node_modules/react-daterange-picker/node_modules/moment``

Whether that is the case seems to depend, as far as I can tell, on:

1. The version of moment the user's project depends on, if it indeed does depend on moment (not unlikely).
2. The order of installation of the dependencies (i.e. the user project's own copy of moment is added later).
3. The version of npm (version 3 has a flattened node_modules by default).

Long story short, when ``react-daterange-picker`` executes ``require("moment")`` either one of these two paths might be imported, and in the worst case, it's not predictable which one:

    ``project_dir/node_modules/react-daterange-picker/node_modules/moment``
    ``project_dir/node_modules/moment``

To make locales work, the user would have to import the desired locale from the correct directory too.

One solution to this might be to make ``moment`` a peer dependency.